src/BuildScriptGenerator/Node/NodeBashBuildSnippet.sh.tpl

```diff
@@ -110,9 +110,11 @@ then
 		cp -f "$SOURCE_DIR/.yarnrc.yml" .
 	fi
 
-	if [ YarnVersionSpec | IsNotBlank ] && [ -f "$SOURCE_DIR/.yarn/releases/yarn-${YarnVersionSpec}"* ]; then
-		cp -f "$SOURCE_DIR/.yarn/releases/yarn-${YarnVersionSpec}"* .
+	{{ if YarnVersionSpec | IsNotBlank }}
```

Comment:
Looks good to me, but I'd still be curious to understand better the change. Was this reproducing every time for this scenario?

Reply:
Every time that we [had prod dependencies and were pruning dev dependencies](https://github.com/microsoft/Oryx/blob/main/src/BuildScriptGenerator/Node/NodeBashBuildSnippet.sh.tpl#L77), we are able to see this error reproduce. But it does not exit the script, and therefore the rest of the script seemed to execute fine

Reply:
Thanks for clarifying!